crates/bevy_log/src/lib.rs

```diff
@@ -218,7 +227,9 @@ impl Plugin for LogPlugin {
             #[cfg(feature = "tracing-tracy")]
             let tracy_layer = tracing_tracy::TracyLayer::default();
 
-            let fmt_layer = tracing_subscriber::fmt::Layer::default().with_writer(std::io::stderr);
+            let mut fmt_layer =
+                tracing_subscriber::fmt::Layer::default().with_writer(std::io::stderr);
+            fmt_layer.set_ansi(self.ansi);
```

Comment:
FYI there is `with_ansi` and that may be more ergonomic. E.g. see this old unmerged PR: https://github.com/bevyengine/bevy/pull/11398/files#diff-ce0ef3820ac31e1eaf71e97972a68d832239ca8affc9441f6dd419f434d15781R179

Reply:
Hmm, `NO_COLOR` did make a difference for the default implementation of Layer I noticed, but I don't easily control the environment variables where my use case is, though I can manually override with `std::env::set_var`
I'm making a 100% Rust iOS app and building it using Xcode (see https://lib.rs/xcode-build-rs), but the logging through Xcode is non-ansi supported and makes a big mess. I'll try `std::env::set_var` ...

Reply:
Yes `std::env::set_var("NO_COLOR", "1");` worked for me, it disabled colours as long as I executed it before I added the `LogPlugin`. I believe propagating an 'implementation detail' like `ansi: bool` is now unnecessary, but I will change this PR to at least document this behaviour so that people after me don't have to go through the same hurdles to disable colour